webpack.config.js

```diff
@@ -13,6 +13,7 @@ Licensed MIT © Zeno Rocha`;
 module.exports = {
   entry: './src/clipboard.js',
   mode: 'production',
+  target: ['web', 'es5'],
```

Comment:
Nice! Did you try with ES5 build to validate the issue opened about it?

Reply:
```
ES-Check: there were no ES version matching errors!  🎉
```